styles/style.css

```diff
@@ -120,25 +123,36 @@ p {
 /* hero */
 
 .hero {
-  background-image: url("header-bike.jpg");
-  background-color: var(--grey-light);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.header-bike {
+  position: relative;
 }
 
 .hero h1 {
-  margin-bottom: 1rem;
+  position: absolute;
+  top: 28%;
   font-size: 2.5rem;
+  color: white;
+  margin-left: 40px;
 }
 
 .hero p {
+  position: absolute;
   font-size: 1.2rem;
-  margin-bottom: 2rem;
+  color: white;
+  top: 35%;
+  margin-left: 40px;
 }
 
-
 /* Headings */
 
 .heading-underline {
   position: relative;
+  margin-top: 30px;
```

Comment:
try to be consistent using `rem` instead of pixel.
rem is preferred as it is more responsive 
Here a reading about it
https://www.digidop.fr/en/blog/rem-pixel-what-a-difference-site